plugins/org.eclipse.fordiac.ide.product/org.eclipse.fordiac.ide.product.target

```diff
@@ -12,8 +12,8 @@
 			<unit id="org.eclipse.elk.libavoid.feature.source.feature.group" version="0.9.1"/>
 		</location>
 		<location includeAllPlatforms="false" includeConfigurePhase="true" includeMode="planner" includeSource="true" type="InstallableUnit">
-			<repository location="https://download.eclipse.org/nebula/releases/latest"/>
-			<unit id="org.eclipse.nebula.feature.feature.group" version="3.1.1.202402011804"/>
+			<repository location="https://download.eclipse.org/nebula/updates/milestone/latest"/>
+			<unit id="org.eclipse.nebula.feature.feature.group" version="3.2.0.202503211629"/>
```

Comment:
please change that to 0.0.0 as we are doing this with gef otherwise with any update of nebual we need to adjust this.